src/FileManager.cpp

```diff
@@ -235,14 +296,14 @@ OperationResult FileManager::deletePath(const QFileInfo &pathInfo)
     }
 
     std::filesystem::path pathToDelete = pathInfo.absoluteFilePath().toStdString();
-
+    QString qPathToDelete = QString::fromStdString(pathToDelete.string());
     // Validate the input path
     if (!isValidPath(pathToDelete))
     {
         return {false, "ERROR: invalid file path." + pathToDelete.filename().string()};
     }
 
-    if (!QFile::moveToTrash(pathToDelete))
+    if (!QFile::moveToTrash(qPathToDelete))
```

Comment:
Good catch. Preapse, you may want to move it closer to where it is used and after the pathToDelete path is validated.

```cpp
QString qPathToDelete = QString::fromStdString(pathToDelete.string());
if (!QFile::moveToTrash(qPathToDelete))
{}
```